cli/fathom_web/train.py

```diff
@@ -290,10 +294,12 @@ def main(training_file, validation_file, stop_early, learning_rate, iterations,
         validation_arg = None
     model = learn(learning_rate, iterations, x, y, validation=validation_arg, stop_early=stop_early, run_comment=full_comment)
     print(pretty_coeffs(model, training_data['header']['featureNames']))
-    accuracy, false_positive, false_negative = accuracy_per_tag(model, x, y)
+    y_pred = model(x)  # Make predictions.
```

Comment:
I actually slightly prefer inlining `model(x)` into the following function call (and the same below, for validation). I don't think it's any harder to comprehend (a look at the name of the param it's getting passed into in accuracy_per_tag() clues you in if not), and it doesn't add more state to an already pretty stateful procedure. What do you think?

Reply:
I'm on board with this.